modules/ting_search/modules/subsearch_translate/subsearch_translate.info

```diff
@@ -0,0 +1,8 @@
+name = Subsearch Translate search
+description = Extend Ting Search with secondary search
```

Comment:
Extend Ting Search with secondary search which translates the search to English and informs user if relevant.